.github/workflows/slash.yml

```diff
@@ -31,7 +31,18 @@ jobs:
       with:
         token: ${{ secrets.LAND_TOKEN }}
         reaction-token: ${{ secrets.LAND_TOKEN }}
-        issue-type: pull-request
-        permission: write
-        commands: |
-          land
\ No newline at end of file
+        config: >
```

Comment:
Specifying the config as JSON allows specifying different parameters for each command